cloudigrade/account/tests/helper.py

```diff
@@ -65,6 +65,49 @@ def generate_aws_instance(account, ec2_instance_id=None, region=None):
     )
 
 
+def generate_single_aws_instance_event(
+        instance, powered_time=None, event_type=None, ec2_ami_id=None,
+        instance_type=None, subnet=None):
+    """
+    Generate single AwsInstanceEvent for testing.
+
+    The ``powered_times`` defines when the instance should be considered
```

Comment:
This comment is incorrect. It references `powered_times` and says it's a tuple, but there is no `powered_times` in this function, and the nearest `powered_time` appears to be a `datetime`.

It looks like this was just blindly copied from `generate_aws_instance_events` but needs to be rewritten to be appropriate for this function.

Reply:
@infinitewarp Good catch! Fixed.